proto/account.proto

```diff
@@ -0,0 +1,6 @@
+syntax = "proto3";
+package account;
```

Comment:
I added a `package` directive to every `.proto`. Without this the codegen will create a file named `_.rs`.

This directive instead adds a file `account.rs` which can be imported and has a name that makes sense.